0_reporting_functions.R

```diff
@@ -431,7 +439,7 @@ ReportGeneration <- function(){
     
     ReportName <- paste0("ClimateAlignmentReport_",strtrim(investor_name_select,20),"_",strtrim(portfolio_name_select,20))
     
-    # if (InvestorType %in% c("InvestorSingle","InvestorMeta")){ReportName <- paste0("ClimateAlignmentReport_",investor_name_select)}
+    if (investor_type %in% c("InvestorSingle","InvestorMeta")){ReportName <- paste0("ClimateAlignmentReport_",investor_name_select)}
     
```

Comment:
this feels like it should be validated

Reply:
does this work? The investor_type is defined in one of the first functions in 4_report_maker so if you've changed it there then that's ok. Otherwise, this line is about naming the output files, so it's not so important.